apps/quiz/models.py

```diff
@@ -1,5 +1,43 @@
-from database import db
+from database.database import db
 
 
 class Quiz(db.Model):
-    pass
+
+    __tablename__ = 'quiz'
+
+    id = db.Column(db.Integer, primary_key=True)
+    category = db.Column(db.String(150), ) 
+    difficulty_level = db.Column(db.String(150))
+    questions = db.relationship('Question', backref='quiz', lazy=True)
+
+    def __init__(self, category, difficulty_level, questions):
+        self.category = category
+        self.difficulty_level = difficulty_level
+
+
+class QuizCategory(db.Model):
+
+    __tablename__ = 'quiz_category'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+
+class Question(db.Model):
+
+    __tablename__ = 'question'
+
+    id = db.Column(db.Integer, primary_key=True)
+    text = db.Column(db.String(500), db.ForeignKey('answer.text'))
```

Comment:
This definietly should not be like this, using others table not primary key as foreign key is quite unwise, will explain this today's evening